app/components/input-field/press-release-publication-channels.hbs

```diff
@@ -4,6 +4,7 @@
     <Auk::Checkbox
       @type="checkbox"
       @checked={{includes publicationChannel @publicationChannels}}
+      @disabled={{if (and (eq publicationChannel this.subscribersFlandersBe) this.disableFlandersBeChannel) true}}
       @label={{publicationChannel.name}}
       {{on "change" (fn this.getSelectedPublicationChannels publicationChannel)}}
```

Comment:
Dit is gepasseerd in een vorige PR, maar `this.updateSelectedPublicationChannels` is misschien een duidelijkere naam voor de action?